zendesk/resource_zendesk_ticket_field.go

```diff
@@ -184,7 +190,59 @@ func resourceZendeskTicketFieldCreate(d *schema.ResourceData, meta interface{})
 	return resourceZendeskTicketFieldRead(d, meta)
 }
 
+func setSchemaFields(d *schema.ResourceData, m map[string]interface{}) error {
```

Comment:
I think this function is useful for other resources. Please move this to utility.go or something like that.